frictionless/helpers.py

```diff
@@ -634,10 +634,62 @@ def dicts_to_markdown_table(dicts: List[dict], **kwargs) -> str:
     return df.where(df.notnull(), None).to_markdown(index=False)
 
 
+# TODO:This is a temporary function to use with tabulate as
+# tabulate 0.8.9 does not support text wrap
+def wrap_text_to_colwidths(list_of_lists: List, colwidths: List = [5, 5, 10, 50]) -> List:
+    """Create new list with wrapped text with different column width.
+    Args:
+        list_of_lists (List): List of lines
+        colwidths (List): width for each column
+
+    Returns:
+        List: list of lines with wrapped text
+
+    """
+    result = []
+    for row in list_of_lists:
+        new_row = []
+        for cell, width in zip(row, colwidths):
+            cell = str(cell)
+            wrapped = textwrap.wrap(cell, width=width)
+            new_row.append("\n".join(wrapped))
+        result.append(new_row)
+    return result
+
+
 def format_bytes(size: int) -> str:
     """Format bytes to larger units"""
     units = ["bytes", "KB", "MB", "GB", "TB"]
     index = math.floor(math.log2(size) / 10)
     if index > len(units):
         index = len(units) - 1
     return units[index]
+
+
+def validation_summary(
```

Comment:
Can we also move it to somewhere in `report.py` (a method or a helper)?

Reply:
Yes I was thinking if we can have this method as a part of `report` or `reportTask`

Reply:
Thank you! I have made it part of report class.